logos/logos-workernode/logos_worker_node/model_cache.py

```
@@ -212,22 +212,35 @@ def _copy_model_sync(self, model_name: str) -> bool:
+            model_name, size_mb, src, partial,
         )
 
         try:
             rsync_available = shutil.which("rsync") is not None
             if rsync_available:
-                result = subprocess.run(  # noqa: S603
-                    ["rsync", "-aL", "--delete", str(src) + "/", str(partial) + "/"],
-                    capture_output=True, text=True,
+                proc = subprocess.Popen(  # noqa: S603
+                    ["rsync", "-aL", "--delete", "--info=progress2", "--no-inc-recursive",
+                     str(src) + "/", str(partial) + "/"],
```

Ruff (0.15.10)

[error] Starting a process with a partial executable path

(S607)